althea/assetlist.json

```diff
@@ -0,0 +1,32 @@
+{
+  "chain_name": "althea",
+  "assets": [
+    {
+      "description": "Althea native token",
+      "denom_units": [
+        {
+          "denom": "aalthea",
+          "exponent": 0
+        },
+        {
+          "denom": "althea",
+          "exponent": 18
+        }
+      ],
+      "base": "aalthea",
+      "name": "Althea Token",
+      "display": "althea",
+      "symbol": "ALTHEA",
+      "logo_URIs": {
+        "png": "https://github.com/chalabi2/althea-appV2/blob/staging/public/althea.png",
```

Comment:
Are you able to create an "images" sub folder and use that path instead?

Reply:
I'll take care of this in another PR